vars/ubuntu-16.04.yml

```diff
@@ -29,5 +29,5 @@ keepalived_uca_repo_dist: "{{ ansible_lsb.codename }}-updates/{{ keepalived_uca_
 keepalived_uca_openstack_release: ocata
 
 ## Prevent start on install
-prevent_start_file: "/usr/sbin/policy-rc.d"
-prevent_start_file_content: "exit 101"
+prevent_start_file: "/etc/systemd/system/keepalived.service"
```

Comment:
We can keep the prevent_start_file as /etc/systemd/system/keepalived.service, but we should probably remove the variable prevent_start_mask_systemd.

We can keep the variable prevent_start_file_content, and replace its value to false, or omit, which would trigger an error if we are entering the loop incorrectly.